Resources/Prototypes/Entities/Markers/Spawners/Random/barsign.yml

```
@@ -0,0 +1,34 @@
+- type: entity
+  name: Bar Sign Spawner
+  id: BarSign #yes this is not a good ID i just cbfa to deal with prototype migrations in maps.
```

I think the ID has to be changed or it will conflict with the old ID that was used for specific bar signs.
```suggestion
  id: RandomBarSign
```

---

this id is the same because it is the same thing. Prior, this prototype was a bar sign that would randomly assign itself one of the different bar sign prototypes. This behavior is kinda weird so it is just being replaced with a spawner, which is how most other things handle this behavior. The conflicts in map files are from old components, not this prototype ID.

---

The old `BarSign` was only random if the `BarSingComponent` didn't specity a specific bar sign. Now it is always random, so the ID no longer makes sense.

---

Given that its just a spawn marker entity, not an actual entity, I think calling it BarSign isn't too confusing. Renaming it would be slightly clearer, but IMO not required.